fastlane_core/lib/fastlane_core/ui/errors/fastlane_error.rb

```diff
@@ -26,12 +26,12 @@ def could_contain_pii?
 
 class Exception
   def fastlane_crash_came_from_custom_action?
-    custom_frame = exception.backtrace.find { |frame| frame.start_with?('actions/') }
```

Comment:
Mh, I don't know, I think to 100% be sure this is available we'd need all 3 checks